### PR TITLE
Split at-rest decryption into its own page

### PR DESCRIPTION
**Pull request by sftim**
Split out from #33285

Add a new page about decrypting an API once you've already set up Kubernetes to encrypt it.

/sig security
/sig auth

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/decrypt-data.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+* You should have some API data that are already encrypted.
+
+{{< version-check >}}
+
+
+<!-- steps -->
+
+## Determine whether encryption at rest is already enabled
+
+By default, the API server uses an `identity` provider that stores plain-text representations
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/decrypt-data.md:55]: nit: Is there a reason to wrap identity in back quotes?

File: content/en/docs/tasks/administer-cluster/decrypt-data.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+that specifies a path to a configuration file. The contents of that file, if you specify one,
+control how Kubernetes API data is encrypted in etcd.
+If it is not specified, you do not have encryption at rest enabled.
+
+The format of that configuration file is YAML, representing a configuration API kind named
+[`EncryptionConfiguration`](/docs/reference/config-api/apiserver-encryption.v1/).
+You can see an example configuration
+in [Encryption at rest configuration](/docs/tasks/administer-cluster/encrypt-data/#understanding-the-encryption-at-rest-configuration).
+
+If `--encryption-provider-config` is set, check which resources (such as `secrets`) are
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/decrypt-data.md:69]: nit: Secrets or secrets?

**Comment by reylejano** [comment]
Preview: https://deploy-preview-42922--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/decrypt-data/

preview looks good
/lgtm

**Pull request merged**